src/training/claim_detection.py

```diff
@@ -59,7 +63,7 @@ def main(
 
 if __name__ == "__main__":
 
-    params = yaml.safe_load(open("params.yaml"))["src.training.claim_detection"]
+    params = yaml.safe_load(open("default_params.yaml"))["src.training.claim_detection_task2"]
```

Comment:
This, again, breaks the usage of this script for claim detection training in task 2. Since we want to run both tasks, but they only differ in these parameters here (right?), it should be enough to add the parameters for task 1 in the default_params.yaml. They weren't added in this PR though.

Also, it is intentional that it is params.yaml here, not default_params.yaml. The idea is that after cloning the repo, you do

cp default_params.yaml params.yaml

and then adjust the parameters in params.yaml as you want - locally! You don't want to always adjust the default params. For example, maybe one of the params is the path to take the data from, and it will be different on different machines. Then you only adjust it in params.yaml. Note that for this reason, params.yaml is added to .gitignore, so whatever changes you make there will be ignored by git.

Reply:
Originally, I didn't know you want a user to make additional steps (such like cp as you said).
So I think it's a typo (default_params.yaml ->params.yaml) when you were writing these codes.

Anyway, it depends on you. So I can delete this branch, right?